modules/ztp-deleting-node-using-siteconfig.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ztp-advanced-install-ztp.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-deleting-node-siteconfig_{context}"]
+= Deleting a node by using the SiteConfig CR
+
+By using a SiteConfig custom resource (CR), you can delete and reprovision a node. 
+This method is more efficient than manually deleting the node.
+
+.Prerequisites
+
+* You have configured the hub cluster to generate the required installation and policy CRs.
+
+* You have created a Git repository in which you can manage your custom site configuration data. The repository must be accessible from the hub cluster and be defined as the source repository for the Argo CD application.
+
+
+.Procedure
+
+. Update the  `SiteConfig` CR to include the `bmac.agent-install.openshift.io/remove-agent-and-node-on-delete=true` annotation:
++
+[source,yaml]
+----
+nodes:
+- hostname: node6
+  role: "worker"
+  crAnnotations:
+    add:
+      BareMetalHost:
+        bmac.agent-install.openshift.io/remove-agent-and-node-on-delete: true
+----
+
+. Suppress the generation of the `BareMetalHost` CR from `SiteConfig`.  
+Update the `SiteConfig` CR to include the `crSuppression.BareMetalHost`, for example:
++
+[source,yaml]
+----
+nodes:
+- hostname: node6
+  role: "worker"
+  crSuppression:
+    - BareMetalHost
+----
+
+. Push the changes to the Git repository and wait for deprovisioning to start. 
+The status of the `BareMetalHost` CR should change to `deprovisioning`. Wait for the `BareMetalHost` to finish deprovisioning, and be fully deleted.
+
+.Verification
+
+. Verify that the `BareMetalHost` and `Agent` CRs for the worker node have been deleted from the hub cluster by running the following commands:
++
+[source,terminal]
+----
+$ oc get bmh -n <cluster-ns>
+----
++
+[source,terminal]
+----
+$ oc get agent -n <cluster-ns>
+----
+
```

Comment:
I think expected outputs can be helpful to a user. Optional if you'd like to include them.

Reply:
Decided against an output here. The verification is to see that something isn't there. Hard to get an output that show the absence of something in a meaningful way.